lxd/openfga/objects.go

```diff
@@ -0,0 +1,45 @@
+package openfga
+
+import (
+	"fmt"
+)
+
+func GroupObject(groupName string) string {
```

Comment:
Please can we change these functions to start with `Object` so they are grouped alphabetically by what type of info they return?

Reply:
Like you've done with the Relation constants.